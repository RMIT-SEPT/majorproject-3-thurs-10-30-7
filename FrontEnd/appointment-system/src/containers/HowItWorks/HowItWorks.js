import React from 'react';
import Layout from '../Layout/Layout';

const HowItWorks = (props) => {
  const redirectRegister = () => {
    props.history.push('/register');
  };

  return (
    <Layout>
      <div>
        <br />
        <h2 className='text-center'>The booking System of your dreams</h2>
        <hr />
        <br />
        <div className='container'>
          <div className='row'>
            <div className='col'>
              <p>
                Are you a company that struggles with the techy side of things?
              </p>
              <p>
                Or maybe you're still stuck with paper notes to keep track of your company's upcoming appointments?
              </p>
              <p>
                Well fortunately for you, Nexus Booking Services makes your life easy!
              </p>
              <p>
                From storing the information of all the employees and customers, quick and easy booking of appointments, and handling of employee availabilities and working times,
                to in-depth statistics about how your company is performing, Nexus Booking Services provides a brilliant foundation for your company to employ booking of services that
                you offer.
              </p>
              <p>
                To get started, simply sign up! Or, if you have questions, visit the Contact Us page.
              </p>
            </div>
          </div>
        </div>
        <div className='d-flex justify-content-center'>
          <button
            className='btn btn-link text-center'
            onClick={redirectRegister}
          >
            Sign Up
          </button>
        </div>
      </div>
    </Layout>
  );
};

export default HowItWorks;
