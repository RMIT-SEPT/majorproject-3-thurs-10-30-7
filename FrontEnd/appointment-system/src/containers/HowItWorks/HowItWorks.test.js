import React from "react";
import { configure, shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import HowItWorks from "./HowItWorks";

configure({ adapter: new Adapter() });

describe("<HowItWorks/>", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<HowItWorks />);
  });

  it("should render contact us page", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
