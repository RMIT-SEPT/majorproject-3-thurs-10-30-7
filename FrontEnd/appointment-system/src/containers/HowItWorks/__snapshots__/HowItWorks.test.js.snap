// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HowItWorks/> should render contact us page 1`] = `ShallowWrapper {}`;
